spec/synapse_pay_rest/http_client_spec.rb

```diff
@@ -0,0 +1,111 @@
+require "spec_helper"
+
+RSpec.describe SynapsePayRest::HTTPClient do
+  let(:client_id) { "e3f19e4bd4022c86e7f2" }
+  let(:client_secret) { "11c94ba6bad74d24a0158bc707f0fc19a86dc08f" }
+  let(:ip_address) { "107.170.246.225" }
+  let(:fingerprint) { "e716990e50b67a1177736960b6357524b22090ccab093d068b3d7a18dbde3f4c" }
```

Comment:
Are the fingerprint, client_id, and client_secret you are putting here just random strings? (let's make sure they are..)